### ISSUE TITLE
brave browser?

### ISSUE THREAD

**Issue by briannyeko** [Fyrd/caniuse#2749]
When will brave browser be added to the testing list?

**Comment by Fyrd** [comment]
Brave uses the Chromium rendering engine, so support should match the latest Chrome. No plans on including it separately.

**Comment by mojoaxel** [comment]
Brave uses electron which e.g. [does not support `window.prompt`](https://github.com/brave/browser-laptop/issues/94).
Still not sure if it make sense to track it Brave separately because of this.

**Comment by briansmith** [comment]
> Brave uses the Chromium rendering engine, so support should match the latest Chrome. No plans on including it separately.

They disable some features, e.g. `<a ping>`, and they plan to disable more. It would be useful to somehow note which features are disabled. Perhaps they could be included as a separate column only for the features where they differ from Chrome?

**Comment by sxiii** [comment]
Chromium-based or electron-based browsers can be, for example, be a part of category inherited by chromium.

**Comment by mbomb007** [comment]
Agreed. Brave isn't Chrome. It's based on Chromium, but can differ. Brave technically has separate market share, and it makes sense for them to be tracked separately.

**Comment by ErrorRaffyline0** [comment]
I don't see why Opera can be supported yet Brave cannot.

**Comment by Mubelotix** [comment]
Brave needs to be added as they support features that do not exist in chrome like IPFS. This is blocking #6151

**Comment by tomasdev** [comment]
Opera has its own rendering engine. It has been historically a browser of its own. Brave is built on top of Chromium and provides features that are not standarized or even drafted (like IPFS mentioned above)

I think it should be Brave's responsibility to provide that "diff". 

The comparison should not be against Opera, but rather against Edge. Edge is also now based on Chromium and I assume it's only there for historical reasons.

**Comment by waynebloss** [comment]
Here are 2 big differences that I've found:

- [No support for Idle Detection API](https://github.com/brave/brave-browser/issues/18269)
  - [Disable idle detection API](https://github.com/brave/brave-core/pull/7054)
- [File System API disabled behind flag](https://github.com/brave/brave-browser/issues/18979)
  - [via](https://github.com/brave/brave-browser/issues/20563)

**Comment by luisalvarado** [comment]
I got here because I was researching why brave does not show on the caniuse but it does ot properly answer why brave is not shown as an option since there are not exactly unique to chrome.

**Comment by Mubelotix** [comment]
> I got here because I was researching why brave does not show on the caniuse but it does ot properly answer why brave is not shown as an option since there are not exactly unique to chrome.

There is no reason, someone just needs to implement it and nobody seems to have invested that time yet

**Comment by luisalvarado** [comment]
> > I got here because I was researching why brave does not show on the caniuse but it does ot properly answer why brave is not shown as an option since there are not exactly unique to chrome.
> 
> There is no reason, someone just needs to implement it and nobody seems to have invested that time yet

Am I able to try? or is it for a specific group?

**Comment by firefoxic** [comment]
Guys, you have a data file of several megabytes! Maybe before adding something else, you should at least drop dead browsers from the database. And all these Brave, Opera, and others would be better grouped into one Chromium-based category, and only somewhere in the notes to the feature explain that it is supported in such and such browser or not.

Sorry if I sound rude, but it is unrealistic to cram the whole zoo of Chromiums into this ark. I went to the repository to look for an issue discussing the rejection of dead browsers so I could maybe vote in favor. And here they want to add more. Even caniuse-lite takes 2 megabytes, although at the start of the project 7 years ago it was less than 1 megabyte. And the vast majority of information (both about dead browsers and very ancient versions of live ones) is hardly ever useful to anyone. It feels like this snowball is going to crush us soon.

**Comment by Mubelotix** [comment]
> Guys, you have a data file of several megabytes! Maybe before adding something else, you should at least drop dead browsers from the database. And all these Brave, Opera, and others would be better grouped into one Chromium-based category, and only somewhere in the notes to the feature explain that it is supported in such and such browser or not.
> 
> Sorry if I sound rude, but it is unrealistic to cram the whole zoo of Chromiums into this ark. I went to the repository to look for an issue discussing the removing of dead browsers so I could maybe vote in favor. And here you want to add more. Even caniuse-lite takes 2 megabytes, although at the start of the project 7 years ago it was less than 1 megabyte. And the vast majority of information (both about dead browsers and very ancient versions of live ones) is hardly ever useful to anyone. It feels like this snowball is going to crush us soon.

You are telling us the design of the project is flawed but I see ways to improve it. Chromium-based browsers all act like chromium except for a few features. That means we only need the information for these highly specific features, which isn't a lot at all. A new format could make this very easy

**Comment by firefoxic** [comment]
> And also, 2 MB isn't going to crush anybody. That's like the size of a small PDF that contains images.

It's not about the current file size, it's about the growth trend. And these megabytes are not the same as a PDF file that just exists. The database has to be constantly maintained. I honestly don't envy Alexis, it's a hell of a lot of work for the good of us all, and I sincerely wish him some relief.

**Comment by pr0cesor** [comment]
> Brave uses the Chromium rendering engine, so support should match the latest Chrome. No plans on including it separately.

Flipper lab website (https://lab.flipper.net/) does not support Brave although as you said it uses chromium rendering engine. So definitely should be checked out why Brave browser is not supported.

**Comment by noliveira95** [comment]
+1 for adding Brave to the list.

**Comment by waynebloss** [comment]
Given that many so-called "progressives" (who migrated to Mastodon where the maintainer's socials are) tend to hate Brendan Eich and the Brave browser... I'd wager that Brave and anything else that Brendan has done is being hard ignored with a passion.

So don't get your hopes up. Clear evidence of engine differences has been shown over and over again here and elsewhere. If you want to fix this glaring omission you'll probably have to fork it and run your own site.

**Comment by Generosus** [comment]
> Brave uses the Chromium rendering engine, so support should match the latest Chrome. No plans on including it separately.

Not quite, amigo!

+ 1 to add Brave to "Can I Use." 

Here's one of many good reasons why:

[Speculative Loading (trending)](https://caniuse.com/mdn-html_elements_script_type_speculationrules) is supported by Chrome, but not by Brave. Brave deprecated the "Preload Pages" settings feature which is required for Speculative Loading. So, in short, Brave does not support Speculative Loading.

Further to the above, Brave is currently undervalued and deserves more credit due to its phenomenal privacy and speed features.

Note: I'm not affiliated with Brave nor its developers :)

Cheers!

**Comment by AlexDeFoc** [comment]
Any more updates? Especially considering the last comment in this issue?

**Comment by luckydonald** [comment]
I mean the diff-to-chrome for chromium-forked browsers totally makes sense.
